lib/lltz_michelson/lltz_michelson.ml

```diff
@@ -10,14 +10,14 @@ module LLTZ = struct
   module P = Lltz_ir.Primitive
 end
 
-module Michelson = struct 
+module Michelson = struct
   module Ast = Michelson.Ast
   module T = Michelson.Ast.Type
 end
 
 module Instruction = Instruction
```

Comment:
?

Reply:
this was most likely done by formatter

Reply:
I don't think the formatter creates code. That alias is useless. Can you try?

Reply:
not really, instead of Michelson.Ast.Type we can just have Michelson.T